<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>142</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>341</width>
    <height>142</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>341</width>
    <height>142</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Telefonía IP - LogIn</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>llamada_reposo.png</normaloff>llamada_reposo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
	border-color: rgb(100, 100, 100);
	border-style: outset;
    border-width: 1px;
    border-radius: 10px;
	/*background-color: rgb(218, 215, 209);*/
	background-color: rgb(240, 240, 240);
}

QLabel#label
{
	background-color: rgb(255, 110, 85);
	border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;

}

QPushButton#init_guardar
{
	background-color: rgb(255, 110, 85);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
  
}



</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_login">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>103</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_salir">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>103</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Nombre:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>72</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Contraseña:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nombre_usuario">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>40</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Nombre de usuario</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_contrasena">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>71</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="maxLength">
     <number>30</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>contraseña</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_mensaje">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Mensaje de Error</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
